### PR TITLE
feat: add DDXV2

### PR DESCRIPTION
**Pull request by andrzejnovak**
- Add DeepDoubleX V2
- Clean up indents

@alefisico

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ cmsrel  CMSSW_10_6_14
 cd  CMSSW_10_6_14/src
 cmsenv
 git cms-addpkg PhysicsTools/NanoAOD
+git cms-merge-topic andrzejnovak:DDXV2_106
```
Comment (alefisico) [inline README.md:18]: Can you include `git cms-addpkg  RecoBTag/Combined` here?
that is needed in the recipe

**Comment by alefisico** [comment]
@andrzejnovak can you check that the latest version has what you need?
http://algomez.web.cern.ch/algomez/testWeb/JMARnanoplusBTag_content_v01p1.html
If yes, after fixing the recipe I can merge it, modify the recipe for the tag and start production

File: README.md (hunk 1 above)
Comment (alefisico) [reply README.md:18]: ah nevermind, there is also a small bug in the submit_all script taht I need to change. I'll merge this PR, and fix this and the submit_all in a diff PR.

**Comment by andrzejnovak** [comment]
@alefisico Ok, thanks a lot!

**Pull request merged**